src/app/accounts/settings.component.html

```diff
@@ -3,7 +3,9 @@
     <div class="modal-content">
       <div class="modal-body form">
         <div class="box">
-          <label class="settingsTitle">{{ "settingsTitle" | i18n: currentUserEmail }} </label>
+          <div class="box-header">
+            {{ "settingsTitle" | i18n: currentUserEmail }}
+          </div>
```

Comment:
This style didn't match other modals. So I updated it to use box-header.

Reply:
Maybe. I can check if that works. I think we use box-header in other areas for the "modal title". Seems like its used whenever there is an existing box. I added modal-title style when no existing boxes are already being used in that area of the modal.

Reply:
I am reviewing these modal/section titles with product designers at the moment. Plan is to do a follow up PR to clean things up across the desktop app.